TravelerApp/TravelerAppCore/Controller/AddHotelInfo.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TravelerAppCore.Models.Hotels;
+
+namespace TravelerAppCore.Controller
+{
+    public static class AddHotelInfo
+    {
+        public static Root CreateNew()
+        {
+            Root saveInfoHotel = new Root()
+            {
+                Reviews = SaveRate(),
+                HotelInfo = SaveInfo()
+            };
+            return saveInfoHotel;
+        }
+        public static HotelInfo SaveInfo()
```

Comment:
zbieranie informacji w projekcie core w taki sposób wprowadza ograniczenie i projekt core wiążemy z aplikacją konsolową. Powiedzmy że teraz zmienia się wymaganie i ta aplikacja ma być aplikacją webową to cała ta funkcjonalność jest to przerobienia :)

Reply:
I tak będzie do przerobienia bo w aplikacji konsolowej nie użyję Console.WriteLine ani Console.ReadLine i tym podobnych :)

Reply:
I tak będzie do przerobienia bo w aplikacji webowej nie użyję Console.WriteLine ani Console.ReadLine i tym podobnych :)